go/mysql/binlog_event.go

```diff
@@ -86,9 +86,20 @@ type BinlogEvent interface {
 	IsWriteRows() bool
 	// IsUpdateRows returns true if this is a UPDATE_ROWS_EVENT.
 	IsUpdateRows() bool
+	// IsPartialUpdateRows returs true if a partial JSON update event
+	// is found. These events are only seen in MySQL 8.0 if the mysqld
+	// instance has binlog_row_value_options=PARTIAL_JSON set.
+	IsPartialUpdateRows() bool
 	// IsDeleteRows returns true if this is a DELETE_ROWS_EVENT.
 	IsDeleteRows() bool
 
+	// IsPseudo is for custom implementations of GTID.
```

Comment:
Please explain the purpose of this Pseudo() function? What are custom implementations of GTID? (Also, I'm not sure I follow the connection to partial JSON. Is this specific for testing?)

Reply:
I don't know the answer to these things either. 🙂 I only moved it to more logically group the `IsBlank` functions together.